intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <rclcpp/rclcpp.hpp>
```

Comment:
I agree that having `memory` further up in the list is preferred. But in some cases (e.g. https://github.com/ros2/demos/pull/80/files#diff-c53d060e7c287a653e8fef96c7f7a4eeR33) that would require quite a few of includes to be changed from `<...>` to `"..."`. Not sure if we want to do that?